### DISCUSSION TITLE
4.4 Patch release

### DISCUSSION THREAD

**Discussion by MichelZ** [abpframework/abp#9855] [Releases]
Hello, can we please have a 4.4 Patch release ASAP? The whole domain resolver bug in Angular (#9728)  thing is keeping me from updating....

Thanks :)

**Comment by maliming** [comment]
hi

I guess it will available next week.

**Reply by 274188A** [reply]
any release notes for 4.4?

**Reply by maliming** [reply]
https://blog.abp.io/abp/ABP.IO-Platform-4.4-Final-Has-Been-Released!

**Reply by 274188A** [reply]
![image](https://user-images.githubusercontent.com/40014477/131938236-6330f26f-87cc-481a-b2cc-7eae7b61b5dd.png)

**Reply by maliming** [reply]
[https://blog.abp.io/abp/ABP.IO-Platform-4.4-Final-Has-Been-Released!](https://blog.abp.io/abp/ABP.IO-Platform-4.4-Final-Has-Been-Released!)
